Scripts/Scenes/Lobby/SceneLobby.cs

```diff
@@ -93,9 +88,7 @@ private async void TimerCountdownCallback()
                 {
                     WebClient.TimerPingMasterServer.Stop();
 
-                    await WebClient.Post("servers/remove", new Dictionary<string, string> {
-                        { "Ip", WebClient.ExternalIp }
-                    });
+                    await WebClient.RemoveAsync();
```

Comment:
`await WebClient.RemoveAsync();` this should be after `await NetworkManager.GameClient.Send(ClientPacketOpcode.Lobby, new CPacketLobby`

Sending the lobby game start packet should take priority and it should be as fast as possible